python_modules/dagster/dagster/core/types/runtime.py

```diff
@@ -227,6 +231,10 @@ def __init__(self, inner_type):
         )
         self.inner_type = inner_type
 
+    @property
+    def display_name(self):
+        return self.inner_type.display_name + '?'
```

Comment:
Ahh this is cool - I think that we may want to leverage this new display_name idea in the configType schema viewer as well. Right now it gets `isList` and uses that to add the `[`, `]`, etc. client side.